recipe/patches/0004-loosen-requirements.patch

```diff
@@ -1,43 +1,42 @@
-From b6371c3abefb0ab2b3e943f43088ab42fac9c5bc Mon Sep 17 00:00:00 2001
+From 504e6239c72e4c5d38b0910796841bdfffa20dc4 Mon Sep 17 00:00:00 2001
 From: "Uwe L. Korn" <uwe.korn@quantco.com>
 Date: Thu, 30 Dec 2021 06:57:09 +0200
-Subject: [PATCH] Fixup requirements
+Subject: [PATCH 4/5] loosen requirements
 
 * Remove unused libclang requirement
 * Remove unused wheel requirement
 * Remove unecessary tensorflow-io-gcs-filesystem
+* Updated for 2.8.0 by ngam
```

Comment:
This patch looks much better now, thanks.

I agree that all the other patches should be dropped (e.g. `git rebase -i tags/v2.8.0`, remove the respective lines, and then another `git format-patch tags/v2.8.0`, which will correct the numbering.

Reply:
done, thanks!